backend/wallet.go

```diff
@@ -521,8 +581,9 @@ func (a *WalletApplication) initTXFromBlockExplorer() error {
 
 // PassKeysToFrontend emits the keys to the settings.Vue component on a
 // 5 second interval
+//TODO - @vito why is 5s interval necessary?
 func (a *WalletApplication) passKeysToFrontend() {
```

Comment:
@grvlle  why is 5s interval necessary

Reply:
It's to reduce the needed CPU cycles to update information on the frontend that barely ever needs to be updated.

Reply:
To prevent unnecessary CPU cycles for data that barely ever has to be updated in the frontend.